src/MerchantCenter/MerchantStatuses.php

```diff
@@ -205,7 +233,6 @@ protected function delete_stale_issues(): void {
 	 * @since x.x.x
 	 */
 	public function clear_product_statuses_cache_and_issues(): void {
-		$this->clear_cache();
```

Comment:
Wonder if the name of the function still valid after removing `clear_product_statuses_cache_and_issues`

Reply:
In part remains valid because we're still deleting any cached intermediate data. I was thinking of renaming it to `clear_product_statuses_intermediate_data_and_issues`. What do you think? Or do you have a better suggestion?

Reply:
I would simplify with `clear_data` so we don't attach too much the name with the logic inside, maybe we lost clarity of what the function does, but with an accurate PHPDOC description, I think is ok. 

However, was just a 💅 / ❓  Not a blocker keeping it like it is